LayoutTests/svg/dom/method-argument-aritychecks.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
Please choose a more descriptive name for the test like `SVGAnimationElement-begin-end-arguments.html` or something similar.